gestion_base_adresse/install/sql/adresse/10_FUNCTION.sql

```diff
@@ -254,6 +254,65 @@ END;
 $$;
 
 
+-- check_impair(integer)
+CREATE FUNCTION adresse.check_impair(nombre integer) RETURNS boolean
+    LANGUAGE plpgsql
+    AS $$
+/* Cette fonction retourne true (vrai) si le nombre passé en argument est impair 
+ 
+   Exemple :  SELECT checkimpair(12);
+              false
+*/
+BEGIN
+ return nombre % 2!=0;
+END;
+$$;
+
+
+-- creation_adresse()
+CREATE FUNCTION adresse.creation_adresse() RETURNS trigger
+    LANGUAGE plpgsql
+    AS $$
+DECLARE
+BEGIN
+
+    NEW.creation_adresse = 'true';
+
```

Comment:
```suggestion
    NEW.creation_adresse = 'true';
```
Par souci d'homogénéité